Ankr-network/game-unity-sdk
# Proxy unity package with an SDK importer code

This PR contains addition of AnkrSDKImporter folder with editor code that checks if the project has AnkrSDK package and its dependency packages imported and if any of those is missing then it finds Packages/manifest.json file and modifies it so that all required packages would be added and also required scoped registry would be added to the same json structure of manifest file. If company and product name of the project corresponds to the original game-unity-sdk project in this case the importing is rejected to not import the duplicate code.

This PR also modifies the PackageExporter class to include AnkrSDKImporter folder to the packages instead of AnkrSDK folder.

How to test:

To test this PR create an AnkrSDKImporter .unitypackage from menu and import it to the newly created project. After it imports all 3 required packages by modifying manifest.json you can try to import the same .unitypackage again to see if it would not repeat the same operation.
